fcrepo-kernel-modeshape/src/main/java/org/fcrepo/kernel/modeshape/FedoraResourceImpl.java

```diff
@@ -408,7 +420,7 @@ private void createTombstone(final Node parent, final String path) throws Reposi
     public Date getCreatedDate() {
         try {
             if (hasProperty(JCR_CREATED)) {
-                return new Date(getProperty(JCR_CREATED).getDate().getTimeInMillis());
+                return new Date(getTimestamp(JCR_CREATED, 0L));
```

Comment:
The `0L` in `getTimestamp(JCR_CREATED, 0L)` seems a little opaque. Would a boolean work?